shared/src/main/java/dice/StatsProcessor.java

```diff
@@ -161,42 +161,44 @@ private StatsData computeStats(RollRecord[] records) {
         double avgRolls = 0.0;
         double avgScore = 0.0;
         double avgNumDiceUsed = 0.0;
-
-        for (int index = 0; index < dbLength; index++) {
-            int temp = 0;
-            RollRecord record = records[index];
-            totalGames = record.getGameId();
-            totalDiceUsed += record.getNumDice();
-        }
-
-        avgNumDiceUsed = (double)totalDiceUsed / (double)dbLength;
-        totalGames = totalGames + 1;
-        avgRolls = (double)totalRolls / (double)totalGames;
-
         int maxGameScore = 0;
-        if (dbLength > 1) {
-            for (int index = 1; index < dbLength; index++) {
-                int[] tempArray = new int[totalGames];
+
+        if (dbLength != 0) {
```

Comment:
This is my attempt at a unit test, though 3 tests are failing now.
`    @Test
    public void getEmptyAllStats() {
        StatsData stats = proc.getAllStats();

        assertThat(stats.getTotalRolls(), equalTo(0));
        assertThat(stats.getAvgRollsPerGame(), equalTo(0.0));
        assertThat(stats.getCumulativeScore(), equalTo(0));
        assertThat(stats.getAvgScore(), equalTo(0.0));
        assertThat(stats.getAvgNumDiceUsed(), equalTo(0.0));
        assertThat(stats.getMaxScore(), equalTo(0));
    }`